docs/source/howto/xps_workchain.ipynb

```diff
@@ -273,6 +269,133 @@
     "print(node.outputs.binding_energies.O_be.get_dict())\n",
     "\n"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run calculation for selected atoms\n",
+    "The previous examples calculated the XPS spectra for selected elements by analyzing the symmetry and finding the non-equivalent site. This is not suitable for large systems with low symmetry, e.g. supported nanoparticles, in which all atoms are non-equivalent. And user usually only interested in the spectra of some special atoms.\n",
+    "\n",
+    "You can use the `atoms_list` input to specify the atoms you are interested in."
```

Comment:
```suggestion
    "## Run calculation for selected atoms\n",
    "The previous examples calculated the XPS spectra for selected elements by analyzing the symmetry and finding all non-equivalent sites. This is not suitable for large systems with low symmetry, e.g. supported nanoparticles, in which all atoms are non-equivalent and the user is usually only interested in the spectra of some special atoms.\n",
    "\n",
    "You can use the `atoms_list` input to specify the atoms you are interested in."
```

Reply:
Just a small correction to how the instructions are worded.

Reply:
Thanks! Done.